frontend/src/components/custom-tools/header/Header.jsx

```diff
@@ -167,13 +170,36 @@ function Header({ setOpenSettings, handleUpdateTool }) {
             />
           </Tooltip>
         </div>
+        {CloneTitle && (
+          <div>
+            <Tooltip title="Clone">
+              <Button
+                icon={<CopyOutlined />}
+                onClick={() => setOpenCloneModal(true)}
+                disbaled={true}
+              />
+            </Tooltip>
+          </div>
+        )}
+        {PromptShareModal && (
+          <div>
+            <Tooltip title="Public Share">
+              <Button
+                icon={<ShareAltOutlined />}
+                disabled={isPublicSource}
+                onClick={() => setOpenShareModal(true)}
+              />
+            </Tooltip>
+          </div>
+        )}
```

Comment:
@harini-venkataraman I think a better approach would be to create a separate component for these two buttons and move it to the cloud repository. This way, you will only need to conditionally import and render the components.